src/server/auth/server/api_server.go

```diff
@@ -533,188 +367,61 @@ func (a *apiServer) Deactivate(ctx context.Context, req *auth.DeactivateRequest)
 		return nil, err
 	}
 
-	// wait until watchAdmins has deactivated auth, so that Deactivate() is less
-	// likely to race with subsequent calls that expect auth to be deactivated.
-	// TODO this is a bit hacky (checking repeatedly in a spin loop) but
-	// Deactivate() is rarely called, and it helps avoid races due to other pachd
-	// pods being out of date.
+	// wait until the clusterRoleBinding watcher sees the deleted role binding,
+	// so that Deactivate() is less likely to race with subsequent calls that
+	// expect auth to be deactivated.
 	if err := backoff.Retry(func() error {
-		if a.isActive() {
+		if err := a.isActive(); err == nil {
 			return errors.Errorf("auth still activated")
 		}
 		return nil
-	}, backoff.RetryEvery(time.Second)); err != nil {
+	}, backoff.RetryEvery(100*time.Millisecond)); err != nil {
```

Comment:
Shorten these timeouts from 1 second to 100 milliseconds, to hopefully reduce test latency.